wagtail_streamfield_migration_toolkit/management/commands/streamdatamigration.py

```diff
@@ -0,0 +1,230 @@
+from functools import lru_cache
+from django.core.management.base import BaseCommand, CommandError
+from django.db import connection
+from django.db.migrations.loader import MigrationLoader
+from django.db.migrations import Migration
+from django.db.migrations.autodetector import MigrationAutodetector
+from django.db.migrations.state import ProjectState
+from django.db.migrations.writer import MigrationWriter
+from django.apps import apps
+from wagtail.blocks import StreamBlock, StructBlock
+
+from wagtail_streamfield_migration_toolkit.migrate_operation import MigrateStreamData
+from wagtail_streamfield_migration_toolkit.operations import (
+    RemoveStreamChildrenOperation,
+    RenameStreamChildrenOperation,
+    RemoveStructChildrenOperation,
+    RenameStructChildrenOperation,
+)
+
+
+class Command(BaseCommand):
+
+    help = "Generate data migrations with the given operations. Note that you can only \
+        generate changes for models in a single app at a time."
+
+    def add_arguments(self, parser):
+        parser.add_argument("--name", help="Use this name for migration file")
+        subparsers = parser.add_subparsers(dest="operation_type")
+
+        rename_parser = subparsers.add_parser("rename", help="rename --help")
```

Comment:
I think we should give the user a short description of the subcommand rather than tell them they can use the `--help` option. For example `help="rename a block"`